FABRIC-AND-CFT.md

```diff
@@ -4,85 +4,85 @@ This page highlights the main differences (both technical and philosophical) bet
 
 ## Cloud Foundation Fabric (a.k.a Fabric, this repo)
 
-Fabric is a collection of Terraform modules and end to end examples meant to be cloned as a single unit and used as is for fast prototyping or decomposed and modified for usage in organizations.
+Fabric is a collection of Terraform modules and end-to-end examples meant to be cloned as a single unit and used as is for fast prototyping or decomposed and modified for usage in organizations. Fabric also provides a robust implementation of foundation [stages](./fast/stages), offering a more complete blueprint for organizations to build upon.
 
 ## Cloud Foundation Toolkit (a.k.a CFT)
 
-CFT is a collection of Terraform modules and examples with opinionated GCP best practices implemented as individual modules for gradual adoption and off the shelf usage in organizations.
+CFT is a collection of Terraform modules and examples with opinionated GCP best practices implemented as individual modules for gradual adoption and off the shelf usage in organizations. CFT also includes an IaC [foundation repository](https://github.com/terraform-google-modules/terraform-example-foundation), though it's less developed compared to Fabric's.
 
 ## Key Differences
 
 <table>
   <tr>
    <td>
    </td>
-   <td><strong>Fabric</strong>
+   <td><strong>Fabric (FAST)</strong>
    </td>
-   <td><strong>CFT</strong>
+   <td><strong>CFT (Cloud Foundation Toolkit)</strong>
    </td>
   </tr>
   <tr>
    <td><strong>Target User</strong>
    </td>
-   <td>Organizations interested in forking, maintaining and customizing Terraform modules.
+   <td>Organizations interested in forking, maintaining, and customizing Terraform modules to have fine-grained control over their infrastructure.
    </td>
-   <td>Organizations interested in using opinionated, prebuilt Terraform modules.
+   <td>Organizations interested in using opinionated, prebuilt Terraform modules with a focus on Google Cloud best practices.
```

Comment:
Fly by comment: Fabric equally follows best practices. I'd say it's even more opinionated on them than CFT.

Reply:
Thanks for pointing this out! You're absolutely right – Fabric (FAST) is also highly focused on Google Cloud best practices, and it might even be considered more opinionated in certain aspects.

Instead of stating one is "more" opinionated than the other, **how about we rephrase to highlight the difference in how they approach opinionation**?

```suggestion
   <td>Organizations seeking prebuilt Terraform modules that offer a curated set of Google Cloud best practices often with a focus on modular composition.
```

Reply:
Fabric is also curated, opinionated, and encourages composition.

> By focusing on how opinionation is expressed rather than making a comparative claim, we can avoid misrepresenting either framework and keep the comparison focused on practical differences.

I don't think the original phrasing misrepresents either framework. The new additions are value judgements and focus on subjective aspects (i.e. x is more opinionated than y)

Reply:
Thanks for the feedback, Julio!

I agree, while not intended as a value judgment, could be interpreted that way. However, I'm also hesitant to introduce potentially new concepts like "modular design" if they're not already established terms in our project's context.

Perhaps the simplest solution is to revert to the original wording for now?

Reply:
Thanks for the feedback, @juliocc!

I agree, while not intended as a value judgment, could be interpreted that way. However, I'm also hesitant to introduce potentially new concepts like "modular design" if they're not already established terms in our project's context.

Perhaps the simplest solution is to revert to the original wording for now?

```suggestion
   <td>Organizations interested in using opinionated, prebuilt Terraform modules.
```